ruleset.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<ruleset name="Yii2 Bootstrap">
+    <exclude-pattern>*/docs/*</exclude-pattern>
+    <exclude-pattern>*/tests/*</exclude-pattern>
+    <exclude-pattern>*/vendor/*</exclude-pattern>
+
+    <rule ref="./vendor/yiisoft/yii2-coding-standards/Yii2/ruleset.xml"/>
+</ruleset>
```

Comment:
@cebe there's no need to put it into CI tools. This ruleset defines what should be covered with codestyle and how it should be treated. At the end of the day it's easier to run `phpcs --standard=ruleset.xml ./` instead of remembering whole configuration with ignored directories.